lib/mapRoutes.js

```diff
@@ -177,3 +193,18 @@ module.exports = function (app) {
 
   return app
 }
+
+function validatePrefix (urlPrefix) {
```

Comment:
Does this handle these three cases and always return the same prefix? I'd recommend expanding the unit tests a little more

* `prefix`
* `/prefix`
* `prefix`
* `/prefix/`
* `prefix/another`
* `/prefix/another`
* `prefix/another/`
* `/prefix/another/`

Reply:
I imagine the context paths could be n levels deep as well.

`/prefix/another/more/.../`

Reply:
Should be all set with those cases. It also works levels deep now. If there is no leading `/` passed such as the `prefix` case one will be appended to the first index of the string.